tests/fuzz/README.md

```diff
@@ -0,0 +1,27 @@
+# Istio fuzzing
+Istio has a series of fuzzers that run continuously through OSS-fuzz.
+
+## Local testing
+To run the fuzzers, first build the fuzzer image from `tests/fuzz`:
+
+```bash
+sudo docker build -t istio-fuzz -f Dockerfile.fuzz .
+```
+Next, get a shell in the container:
+```bash
+sudo docker run -it istio-fuzz
+```
+At this point, you can navigate to `tests/fuzz` and build any of the fuzzers:
+
+```bash
+cd $PATH_TO_FUZZER
+go-fuzz-build -libfuzzer -func=FUZZ_NAME && \
+clang -fsanitize=fuzzer PACKAGE_NAME.a -o fuzzer
```

Comment:
Is there a major difference between libfuzzer and running `go-fuzz` directly?

Reply:
Both engines will be fine for local testing.